isovar/variant_read.py

```diff
@@ -148,8 +150,13 @@ def variant_read_from_single_read_at_locus(read, ref, alt):
                     read))
             return None
 
+    if sequence[read_pos_before + 1:read_pos_after] != alt:
```

Comment:
Not sure I understand? Are you saying that neither read would agree with the variant then and you'd back to 0 `VariantRead`? Is that what happens now or what you'd want to happen?